src/main/java/bio/terra/service/filedata/azure/tables/TableFileDao.java

```diff
@@ -56,13 +57,14 @@ public void createFileMetadata(TableServiceClient tableServiceClient, FireStoreF
 
   public boolean deleteFileMetadata(TableServiceClient tableServiceClient, String fileId) {
     TableClient tableClient = tableServiceClient.getTableClient(TABLE_NAME);
-    TableEntity entity = tableClient.getEntity(PARTITION_KEY, fileId);
-    if (entity == null) {
+    try {
+      logger.info("deleting file metadata for fileId {}", fileId);
+      TableEntity entity = tableClient.getEntity(PARTITION_KEY, fileId);
+      tableClient.deleteEntity(entity);
+      return true;
+    } catch (TableServiceException ex) {
```

Comment:
Sorry, do you think you could add a quick log statement here?  Just because we just ran into a similar thing yesterday when we were dong that walkthrough with michael.  Just to capture the exception